### PR TITLE
Initial version of hack - always preserve rdfs:class if present.

### PR DESCRIPTION
**Pull request by sesuncedu**
**DO NOT MERGE**
Test cases failing due to class cast error 
Control via option not done.

### CODE REVIEW

File: api/src/main/java/org/semanticweb/owlapi/io/RDFParserMetaData.java [hunk 1]
```diff
@@ -39,21 +39,22 @@
...
+ *        the set of triples not parsed
      * @param guessedDeclarations
-     *        guessed declarations map
+     * @param rdfsClassDeclarations
      */
     public RDFParserMetaData(@Nonnull RDFOntologyHeaderStatus headerStatus,
-            int tripleCount, @Nonnull Set<RDFTriple> unparsedTriples,
-            @Nonnull ArrayListMultimap<IRI, Class<?>> guessedDeclarations) {
+                             int tripleCount, @Nonnull Set<RDFTriple> unparsedTriples,
+                             @Nonnull ArrayListMultimap<IRI, Class<?>> guessedDeclarations, Set<IRI> rdfsClassDeclarations) {
```
Comment (ansell) [inline api/src/main/java/org/semanticweb/owlapi/io/RDFParserMetaData.java]: Should we add another version of the constructor that preserves the previous signature and sends an empty set here? That would help preserve backwards compatibility if we have this class as part of our public API.

**Comment by ansell** [comment]
Looks good to me except for the one comment about constructor signatures.

File: api/src/main/java/org/semanticweb/owlapi/io/RDFParserMetaData.java (hunk 1 above)
Comment (sesuncedu) [inline api/src/main/java/org/semanticweb/owlapi/io/RDFParserMetaData.java]: Definitely. Also, I hate having all this stuff in a class that is part of a public API.

**Comment by sesuncedu** [comment]
Needs config settings.
I really wish we could use the rio config classes, but that's too much for
this ugly a hack.

**Comment by ansell** [comment]
Using the Rio config style with named and typed parameters (if not the actual RioConfig class) may be useful to target for version 5.

**Pull request closed without merging**